<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>601</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addCityButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add City</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRouteButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Route</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityNameInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="city1Input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="city2Input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="distanceInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="algorithmStepsTextEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>109</y>
      <width>201</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
